api/ImageData.json

```diff
@@ -77,10 +77,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "7"
```

Comment:
This is a constructor and I don't trust our process for constructors given two recent cases of incorrectly inferring false. Can you verify this manually?

Reply:
I've filed https://github.com/foolip/mdn-bcd-collector/issues/883 about those doubts.

Reply:
Not sure why "7" was produced here; the constructor test is working, hmm...  I checked in Safari 6.1 with the collector, and it returns `true`.  Correcting.